flujo/recipes/factories.py

```
@@ -411,17 +411,60 @@
+        pass
+
+    # Robust fallbacks sourced from recorded step history to avoid CI-only edge cases
+    # 1) Prefer the validator step's output for the checklist
+    if checklist is None:
+        for step in reversed(result.step_history):
+            try:
+                if isinstance(step.output, Checklist):
+                    checklist = step.output
+                    break
+            except Exception:
+                continue
```

Ruff (0.12.2)

`try`-`except`-`continue` detected, consider logging the exception

(S112)